Source/WebKit/UIProcess/win/PageClientImpl.cpp

```diff
@@ -232,6 +240,63 @@ void PageClientImpl::preferencesDidChange()
     notImplemented();
 }
 
+bool PageClientImpl::handleRunOpenPanel(WebPageProxy*, WebFrameProxy*, const FrameInfoData&, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
+{
+    ASSERT(parameters);
+    ASSERT(listener);
+
+    HWND viewWindow = viewWidget();
+    if (!IsWindow(viewWindow))
+        return false;
+
+    bool isAllowMultipleFiles = parameters->allowMultipleFiles();
+    Vector<wchar_t> fileBuf(isAllowMultipleFiles ? maxFilePathsListSize : MAX_PATH);
+
+    OPENFILENAME ofn { };
+
+    // Need to zero out the first char of fileBuf so GetOpenFileName doesn't think it's an initialization string
+    fileBuf[0] = '\0';
+
+    ofn.lStructSize = sizeof(ofn);
+    ofn.hwndOwner = viewWindow;
+    ofn.lpstrFile = fileBuf.data();
+    ofn.nMaxFile = fileBuf.size();
+    auto dialogTitle = uploadFileText();
+    auto dialogTitleCharacters = dialogTitle.wideCharacters(); // Retain buffer long enough to make the GetOpenFileName call
+    ofn.lpstrTitle = dialogTitleCharacters.data();
+    ofn.Flags = OFN_ENABLESIZING | OFN_FILEMUSTEXIST | OFN_PATHMUSTEXIST | OFN_EXPLORER;
```

Comment:
That's right. Both OFN_ENABLESIZING and OFN_EXPLORER are unnecessary. I'll remove them.

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/commdlg/ns-commdlg-openfilenamea

OFN_ENABLESIZING is unnecessary as you say.
```
By default, the Explorer-style Open and Save As dialog boxes allow the dialog box to be resized regardless of whether this flag is set. 
```

If OFN_EXPLORER is not set, the old-style dialog that looks like Windows 3.1 will be shown.
```
OFN_ALLOWMULTISELECT 0x00000200
The File Name list box allows multiple selections. If you also set the OFN_EXPLORER flag, the dialog box uses the Explorer-style user interface; otherwise, it uses the old-style user interface.
```

Then I'll remove OFN_ENABLESIZING.

Reply:
Fixed at https://github.com/WebKit/WebKit/commit/70ce9358458f9401040946ef3392f2ae55d6fa3e
Removed only OFN_ENABLESIZING.